### PR TITLE
Use core-next 0.1.0-M2

### PR DESCRIPTION
**Pull request by eed3si9n**
/review @havocp, @jsuereth

### CODE REVIEW

File: project/Dependencies.scala [hunk 1]
```diff
@@ -46,6 +46,9 @@ object Dependencies {
   val sbtCompilerInterface = sbtOrg % "interface" % sbtMainVersion
   val sbtCompletion        = sbtOrg % "completion" % sbtMainVersion
 
+  val coreNextVersion      = "0.1.0-M1"
+  val coreNext             = Def.setting { Defaults.sbtPluginExtra(sbtOrg % "core-next" % coreNextVersion, sbtBinaryVersion.value, scalaBinaryVersion.value) }
```
Comment (havocp) [inline project/Dependencies.scala]: is this the library or the plugin? should be the library only right

Comment (eed3si9n) [inline project/Dependencies.scala]: It's a library that uses sbt's code base, so I'm currently shipping it as a plugin.

Comment (jsuereth) [inline project/Dependencies.scala]: you can;t because the launcher wasn't resolving this correctly and classpaths get hosed when you try integration tests

**Comment by jsuereth** [comment]
LGTM

**Pull request merged**